BearerBot/README.md

```diff
@@ -0,0 +1,190 @@
+# Bearer Token Bot
```

Comment:
The `BearerBot` folder exists both at the top level of this repo and also within the 'faas-samples' folder. As far as I can tell, the JS files are identical, however, the readmes differ in some areas. Could we ensure that the correct README is in the faas-samples folder and delete the BearerBot folder from the top level?